openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -125,17 +135,34 @@ def _update_tool_with_data(self, tool, data):
         original_subset = tool.GetData("openpype.subset")
         subset = data["subset"]
         if original_subset != subset:
-            # Subset change detected
-            # Update output filepath
-            workdir = os.path.normpath(legacy_io.Session["AVALON_WORKDIR"])
-            filename = f"{subset}..exr"
-            filepath = os.path.join(workdir, "render", subset, filename)
-            tool["Clip"] = filepath
-
-            # Rename tool
-            if tool.Name != subset:
-                print(f"Renaming {tool.Name} -> {subset}")
-                tool.SetAttrs({"TOOLS_Name": subset})
+            self._configure_saver_tool(data, tool, subset)
+
+    def _configure_saver_tool(self, data, tool, subset):
+        formatting_data = deepcopy(data)
+        self.log.warning(pformat(formatting_data))
```

Comment:
Would be good to pull an updated branch of https://github.com/ynput/OpenPype/pull/4943 so this is removed.